api/src/test/scala/dcos/metronome/api/v1/controllers/JobSpecControllerTest.scala

```diff
@@ -45,6 +45,30 @@ class JobSpecControllerTest extends PlaySpec with OneAppPerTestWithComponents[Mo
       contentType(response) mustBe Some("application/json")
     }
 
+    "creates job when sending docker spec with forcePullImage property against v0" in {
+      Given("Job spec with forcePullImage")
+      val specJson = "{\"id\":\"spec1\",\"run\":{\"cpus\":1,\"mem\":128,\"disk\":0,\"docker\":{\"image\":\"image\", \"forcePullImage\": true}}}"
```

Comment:
Maybe it is worth using triple quotes here? Like
```
@ """{"foo":"bar"}""" 
res0: String = "{\"foo\":\"bar\"}"
```

Reply:
I was stay consistent... and truth be known, I copied a previous test and modified for this purpose... but yeah... this should be the way